### PR TITLE
Update DevX ownership in our team homepage

### PR DESCRIPTION
**Pull request by jhchabran**
First batch of changes to update our ownership.

### CODE REVIEW

**Comment by davejrt** [comment]
LGTM

One point, are we also responsible for admin of cloud/dogfood? User management, etc?

**Review by davejrt — APPROVED** [review]

File: content/departments/engineering/teams/dev-experience/index.md [hunk 1]
```diff
@@ -21,6 +21,17 @@ Find out more about the Dev Experience team's mission, vision, and strategic pla
...
   - [CLA bot](https://github.com/sourcegraph/clabot-config)
   - [Monitoring generator](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/monitoring)
+  - [Release Tool](../../dev/tools/release.md)
+- [DotCom](https://sourcegraph.com/search)
+  - Monitoring
+  - Analytics
+  - Deployments
+  - Cloudflare (with the exclusion of the WAF)
+  - SOC2 control points partially pertaining to DotCom: `CI-86`, `CI-100` and `CI-108`.
+- [DogFood](https://k8s.sgdev.org/)
```
Comment (bobheadxi) [inline content/departments/engineering/teams/dev-experience/index.md:31]:
```suggestion
- [Dogfood](https://handbook.sourcegraph.com/departments/engineering/dev/process/deployments/instances/#k8ssgdevorg)
```

File: content/departments/engineering/teams/dev-experience/index.md [hunk 2]
```diff
@@ -21,6 +21,17 @@ Find out more about the Dev Experience team's mission, vision, and strategic pla
   - [`sg` - the Sourcegraph developer tool](https://docs.sourcegraph.com/dev/background-information/sg)
   - [CLA bot](https://github.com/sourcegraph/clabot-config)
   - [Monitoring generator](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/monitoring)
+  - [Release Tool](../../dev/tools/release.md)
+- [DotCom](https://sourcegraph.com/search)
```
Comment (bobheadxi) [inline content/departments/engineering/teams/dev-experience/index.md:25]:
```suggestion
- [DotCom](https://handbook.sourcegraph.com/departments/engineering/dev/process/deployments/instances/#sourcegraph-cloud)
```

**Comment by marekweb** [comment]
This PR predates the migration from `engineering_ownership.yml` to the ownership markdown page. Is the rest of the PR out of date too or do we want to merge the changes to the team page?

**Comment by jhchabran** [comment]
> This PR predates the migration from `engineering_ownership.yml` to the ownership markdown page. Is the rest of the PR out of date too or do we want to merge the changes to the team page?

That's a good question, thanks for bringing this forward. We can probably take a look with @quinnhare  to make sure everything is correct 🙏

**Pull request closed without merging**